custom_components/wyzeapi/light.py

```diff
@@ -172,16 +176,10 @@ def extra_state_attributes(self):
             ATTR_ATTRIBUTION: ATTRIBUTION,
             "state": self.is_on,
             "available": self.available,
-            "device model": self._bulb.product_model,
+            "device_model": self._bulb.product_model,
```

Comment:
Brings all the attributes in line with HA naming conventions. Could be a breaking change, but I doubt anybody’s relying on this.

Reply:
We will have to test but I don't think it would cause any problems.